perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/ResourcesManagerBlImpl.java

```diff
@@ -1079,6 +1081,74 @@ public EnrichedResource filterOnlyAllowedAttributes(PerunSession sess, EnrichedR
 		return assignedGroups;
 	}
 
+	@Override
+	public void activateGroupResourceAssignment(PerunSession sess, Group group, Resource resource, boolean async) throws WrongReferenceAttributeValueException, GroupResourceMismatchException, WrongAttributeValueException, GroupNotDefinedOnResourceException {
+		GroupResourceStatus status = getResourcesManagerImpl().getGroupResourceStatus(sess, group, resource);
+		if (status == GroupResourceStatus.ACTIVE || status == GroupResourceStatus.PROCESSING) {
+			// silently skip
```

Comment:
I think in the case of `PROCESSING` we don't want to skip the validation. It might not have even started and we might want to run it as soon as possible.

Reply:
I removed `PROCESSING` from the if condition.